scripts/release/collect_license_files.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
Is this file copy from other Apache project ? If so, we may need point it out in LICENSE.